spring-cloud-dataflow-rest-resource/src/main/java/org/springframework/cloud/dataflow/rest/resource/AppMetricResource.java

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.dataflow.rest.resource;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.springframework.hateoas.ResourceSupport;
+
+/**
+ * Rest resources for runtime metrics response.
+ *
+ * @author Janne Valkealahti
+ *
+ */
+public class AppMetricResource extends ResourceSupport {
+
+	private MetricsHolder entity;
+
+	/**
+	 * Instantiates a new app metric resource.
+	 *
+	 * @param entity the entity
+	 */
+	public AppMetricResource(MetricsHolder entity) {
```

Comment:
Other ResourceSupport subclasses we have contain a private or protected constructor with comments related to serialization.  This may need to be added.  @ghillert ?